src/pages/TeamPage/index.js

```diff
@@ -0,0 +1,348 @@
+import React, { useState, useEffect } from "react";
+import {
+  Button,
+  Typography,
+  Box,
+  Container,
+  Stack,
+  CssBaseline,
+  Card,
+} from "@mui/material";
+import { Grid } from "@mui/material";
+import useMediaQuery from "@mui/material/useMediaQuery";
+import useStyles from "./style";
+import axios from "axios";
+import Tippy from "@tippyjs/react";
+import "./style.css";
+import { breakpoints } from "../../theme/constant";
+import LinkedIn from "../../components/common/SocialMediaIcons/LinkedIn";
+import Twitter from "../../components/common/SocialMediaIcons/Twitter";
+
+function shuffleObject(obj) {
+  let newObj = {};
+  var keys = Object.keys(obj);
+  keys.sort(function (a, b) {
+    return Math.random() - 0.5;
+  });
+  keys.forEach(function (k) {
+    newObj[k] = obj[k];
+  });
+  return newObj;
+}
+
+function TeamPage() {
+  useEffect(() => {
+    axios({
+      url: `https://anandpatel504.github.io/tarabai-shinde/data/ng_team.json`,
+    }).then((res) => {
+      setTeam(res.data);
+    });
+  }, []);
+  const [team, setTeam] = useState([]);
+  const [members, setMembers] = useState({
+    teamMembers: true,
+    volunteers: false,
+  });
+  const classes = useStyles();
+  const isActive = useMediaQuery("(max-width:" + breakpoints.values.sm + "px)");
+  const isActiveIpad = useMediaQuery("(max-width:1300px)");
+  // console.log(breakpoints.values.sm);
+  console.log(isActive);
+  function Popup(props) {
```

Comment:
When you mouse over someone's picture on a browser whose width is greater than 600px, you don't see any text because it's white and the background is white.  The names below people's pictures are also white and are not aligned over their titles.

Reply:
Actually popup comes but it overflows to right as we around 600px when we switch it to mobile and reload the browser it will come below the name .
[
![image](https://user-images.githubusercontent.com/66902288/164084407-1a105abf-ae39-4856-850e-a76db32e297b.png)
](url)

Reply:
@bunnie26 This is what I'm seeing:
![image](https://user-images.githubusercontent.com/9159008/164111262-2b46b0b7-e89f-4962-a2c5-f5fcd9ef8f94.png)

When selecting the text, I see this:
![image](https://user-images.githubusercontent.com/9159008/164111397-0ef6c02b-82ff-47e3-bcd1-c02c70a04714.png)

When inspecting the element, the problematic generated CSS rule is:
```
.jss9 {
    color: white;
    text-decoration: none;
}
```
And if I remove the `color: white` declaration, I see this as I should (the alignment seems to be good on this run):
![image](https://user-images.githubusercontent.com/9159008/164111924-50e538ba-8e48-47f2-938f-9454fa03a4fb.png)